scripts/run_test_locally.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -l
```

Comment:
Can we make this work as an option to `run_tests.sh`? eg. `LOCAL=true ./scripts/run_tests.sh`

It feels like 99% of the logic should be the same so let's try and avoid having two scripts that do things differently / can get out of sync.